scalingo/resource_scalingo_scm_integration.go

```diff
@@ -115,9 +115,7 @@ func resourceScmIntegrationDelete(ctx context.Context, d *schema.ResourceData, m
 }
 
 func resourceScmIntegrationImport(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
-	d.SetId(d.Id())
-
-	diags := resourceGithubLinkRead(ctx, d, meta)
```

Comment:
I must say that I have no idea what this change does, but I am happy to validate it ^^

Joke aside, if you are sure about what it does and that it works, LGTM